calico.go

```diff
@@ -316,9 +316,9 @@ func cmdAdd(args *skel.CmdArgs) error {
 			fmt.Fprintf(os.Stderr, "Calico CNI creating profile: %s\n", conf.Name)
 			var inboundRules []api.Rule
 			if wepIDs.Orchestrator == "k8s" {
-				inboundRules = []api.Rule{{Action: "allow"}}
+				inboundRules = []api.Rule{{Action: "Allow"}}
```

Comment:
Really should use `api.Allow`... but I'll Allow it.

Reply:
Unless you want to fix it up, in which case I'd be really happy.

Reply:
Need to rev libcalico-go once it's ready any, so I'll update that as well!